Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM openjdk:8-alpine as builder
```

Comment:
We should switch to a more docker friendly version of the jdk/jre images.
Smaller footprint. Better jvm memory consumption.
I started playing around with the latest version from adoptopenjdk which works very well.
See https://github.com/ancho/jbake/blob/feature/docker-multistage-build/Dockerfile

Reply:
Yeah happy to go with adoptopenjdk, I'll update now.

Reply:
Had to revert to adoptopenjdk/openjdk11 as Gradle 6.8.3 didn't like openjdk16, but noticed your branch had an update to Gradle 7 so we can upgrade to openjdk16 at the same time.